src/styles/global.css

```
@@ -282,28 +279,27 @@ html {
+    display: block;
+  }
+}
 
 dialog {
-    scale: 0;
+  scale: 0;
 
-    transition: scale .4s ease, display .4s ease allow-discrete;
+  transition:
+    scale 0.4s ease,
+    display 0.4s ease allow-discrete;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)